src/Core/Scheduler/Scheduler.js

```diff
@@ -119,11 +119,20 @@ Scheduler.prototype.runCommand = function runCommand(command, queue, executingCo
 Scheduler.prototype.execute = function execute(command) {
     // TODO: check for mandatory commands fields
 
-
     // parse host
     const layer = command.layer;
+    let host;
 
-    const host = layer.url ? new URL(layer.url).host : undefined;
+    if (layer.url) {
```

Comment:
no actually, what we meant is 
```js
const host = layer.url ? new URL(layer.url, document.location).host : undefined;
```
This should work in all cases.

Reply:
updated